data/dbus/snapd.conf

```diff
@@ -0,0 +1,3 @@
+<busconfig>
```

Comment:
That's already the case with pretty much all of the security backends, and we actually already have a dbus backend that is tasked precisely with writing out that sort of file, and we ought to be removing these already when the package goes away (are we?).